### ISSUE TITLE
vmapkey custom bindings throwing error now

### ISSUE THREAD

**Issue by paradox460** [brookhong/Surfingkeys#2150]
## Error details

SurfingKeys: 1.17.0

Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36

URL: <The_URL_Where_You_Find_The_Issue>

## Context

I've had some vmapkey custom bindings in my configuration, and today they started throwing the following error:

```
[SurfingKeys] Error found in settings: TypeError: vmapkey is not a function
```

**Comment by elifoglu** [comment]
Having the same issue, it just started today. Same version, and,

Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36

**Comment by brookhong** [comment]
Known issue, will be fixed soon, thanks.

**Comment by brookhong** [comment]
Fixed in 1.17.1.
